src/Item/Item.types.ts

```diff
@@ -63,9 +63,16 @@ export type FullItem = Omit<ItemAttributes, 'urn_suffix'> & {
 
 export type DBItemApprovalData = Pick<
   ItemAttributes,
-  'id' | 'urn_suffix' | 'local_content_hash'
+  'id' | 'local_content_hash'
 >
-export type ItemApprovalData = Pick<FullItem, 'urn'> & { content_hash: string }
+export type ItemApprovalData = {
+  cheque: {
```

Comment:
It bothers me that I can't really type this the same as `PublishCheque`

```ts
export type PublishCheque = {
  signedMessage: string
  signature: string
  qty: number
  salt: string
}

// or maybe as SlotUsageChequeAttributes
```

but I can't seem to actually generalize it

Reply:
What about using `cheque: Omit<PublishCheque, 'signedMessage'>` ? I'd be OK with that :)